Documentation/GettingStarted.md

```diff
@@ -727,6 +727,36 @@ You can recover from failure of observable by using `catch` operator. There are
 
 There is also `retry` operator that enables retries in case of errored sequence.
 
+### Default error handling
+
+If you don't provide `onError` callback default handling will be performed. `Hooks` enum is responsible for it. 
+
+It just prints received error in `DEBUG` mode and do nothing in`RELEASE`. However, you can add additional configuration to this behavior.
+
+In order to enable detailed callstack logging set `Hooks.recordCallStackOnError` flag.
+```swift
+    /* add somewhere in
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil)
+    */
+    Hooks.recordCallStackOnError = true
+    ...
+    Observable.just(error).subscribe(onNext: { // detailed callstack will be printed out
+        print($0)
+    })
+
+```
+
+Implement `Hooks.customCaptureSubscriptionCallstack` closure, if you need custom way of grabbing logs.
+
+Implement `Hooks.defaultErrorHandler` if you need custom reaction on unhandled errors (e.g. send stacktrace to analytics). 
+
+As error logs are quite inconspicuous, you might want raise fatal error if unhandled error occured. There is already `bind(onNext)` method for that kind of needs.
```

Comment:
This specific section is quite opinionated and I’d prefer to remove it. usage of bind on next is reserved to the context of dealing with UI and it’s not just a way to capture errors every where.

Reply:
If you say that, I will remove it :)
But could u please explain to me, how potential user should understand that? There are no docs about "reserved to the context of dealing with UI". It's just a wrapper which calls rxFatalErrorInDebug in `onError` block. Why can't I use it not only for UI, if I just want to get this "fatalError" behavior?

Maybe I will add an additional note like "bind(onNext:) is reserved only for UI"? What do you think?

Reply:
Because `subscribe` is for subscribing to a stream and `bind` is always used to create some connection. I guess it's not really "reserved for UI" but it's where you would most often use it. I don't think it's something that needs documentation, it's just an API.

Reply:
My point was that its entirely not related to documenting Hooks so I would prefer not having it there personally.

Reply:
Sure, get it. Delete this section.